test/libsolidity/semanticTests/errors/small_error_optimization.sol

```diff
@@ -0,0 +1,22 @@
+error E();
+contract A {
+	uint8[] x;
+	function f() public {
+		for (uint i = 0; i < 100; ++i)
+			x.push(uint8(i));
+		revert E();
+	}
+}
+contract B {
+	function f() public {
+		(new A()).f();
+	}
+}
+// ----
+// f() -> FAILURE, hex"92bbf6e8"
+// gas irOptimized: 221918
+// gas irOptimized code: 42800
+// gas legacy: 233746
+// gas legacy code: 37400
+// gas legacyOptimized: 224864
+// gas legacyOptimized code: 34200
```

Comment:
Is this really the cost before optimization as the description says? i don't see it being updated in the other commit so if this was the case, CI would be failing.

Reply:
No, it was my bad, I ended up fixing up the first commit instead of the second and then squashed. This was the before and after:
```
  Running via Yul: 
  Expected result:
  // f() -> FAILURE, hex"92bbf6e8"
  // gas irOptimized: 222533
  // gas irOptimized code: 43200
  // gas legacy: 233746
  // gas legacy code: 37400
  // gas legacyOptimized: 224864
  // gas legacyOptimized code: 34200

  Obtained result:
  // f() -> FAILURE, hex"92bbf6e8"
  // gas irOptimized: 221918 [-615 (+0%)]
  // gas irOptimized code: 42800 [-400 (+0%)]
  // gas legacy: 233746
  // gas legacy code: 37400
  // gas legacyOptimized: 224864
  // gas legacyOptimized code: 34200
```

Reply:
ok. That's a nice improvement.

But you should still at least update the commit description if you're not going to fix it in the commit. Or just squash it, because if it's wrong, it's worse than useless.

Reply:
Agreed. I'll just squash everything into one comment, and leave this comment here in case someone is curious and stumbles upon it.